src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,61 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.addressbook.data.person.Person;
+
+public class UtilsTest {
+    
+    private Utils utilsTest;
+    
+    @Before 
+    public void setup() {
+        utilsTest = new Utils();
+    }
+    
+    @Test
+    public void nullCheckerForNullItems() {
```

Comment:
Note that we have a special naming convention for test method names. Please refer coding standard for more details